TracyC.h

```diff
@@ -73,6 +73,11 @@ typedef const void* TracyCZoneCtx;
 #define TracyCMessageCS(x,y,z,w)
 #define TracyCMessageLCS(x,y,z)
 
+#ifdef TRACY_FIBERS
+#  define TracyFiberEnter(fiber)
+#  define TracyFiberLeave
```

Comment:
These functions (and the second set below) should probably be named `TracyCFiber...`.

Reply:
Whoops... Not sure how I missed that. Should be updated now.